engine/communication/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "communication"
+version = "0.1.0"
+authors = ["Elena Frank <elena.frank@iota.org>"]
+edition = "2018"
+license = "Apache-2.0 OR MIT"
+readme = "README.md"
+
+[dependencies]
+async-std = "1.6.2"
+async-trait = "0.1.40"
+bytes = "0.5.6"
+clap = { version = "3.0.0-beta.1", features = ["yaml"] }
+futures = "0.3.1"
+libp2p = {version = "0.28.1", default-features = false, features = ["dns", "identify", "mdns-async-std", "mplex", "noise", "request-response", "tcp-async-std", "yamux", "websocket"]}
+prost = {version = "0.6.1", default-features = false, features = ["prost-derive"] }
+regex = "1.3.9"
+thiserror = "1.0.21"
+serde = { version = "1.0.117", default-features = false, features = ["alloc", "derive"] }
+serde_json = { version = "1.0.59", default-features = false, features = ["alloc"] }
+riker = "0.4.1"
```

Comment:
Riker 0.4.2 was just published, so this is now out of date. In order to stay 'synced' with the client deps, please drop the patch versions for now. At some point we may return to this, but I am concerned that things stop working in concert if we don't have parity.